substrate/frame/staking/src/weights.rs

```diff
@@ -1,49 +1,23 @@
-// This file is part of Substrate.
```

Comment:
Instead of regenerating the whole file, please only add the newly generated `bond_to` benchmark to the old file. It will make upstream updates easier in the future.